examples/website/data-filter/range-input.js

```diff
@@ -1,36 +1,37 @@
 /* global requestAnimationFrame, cancelAnimationFrame */
 import React, {PureComponent} from 'react';
-import {Client as Styletron} from 'styletron-engine-atomic';
-import {Provider as StyletronProvider} from 'styletron-react';
-import {LightTheme, BaseProvider, styled} from 'baseui';
+import {styled} from 'baseui';
```

Comment:
A bit weird to switch between styletron and styled in the examples. Should we choose one? Maybe styled?

Reply:
From my understanding, baseui styled is a re-exported version of Styletron's styled (see https://baseweb.design/components/styled/).  This change doesn't switch to a different styling system, it just moves the provider setup out of the UI component up to the root of the application (see my above comment).